src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -190,6 +196,9 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         this.plagiarismCaseService.getPlagiarismCaseInfoForStudent(this.courseId, this.exerciseId).subscribe((res: HttpResponse<PlagiarismCaseInfo>) => {
             this.plagiarismCaseInfo = res.body ?? undefined;
         });
+        this.sessionService.getCurrentSession(this.exerciseId).subscribe((session: HttpResponse<IrisSession>) => {
```

Comment:
What happens if no session exists? This would return a 404, which should probably be silenced.

Reply:
@Hialus should the server create a new session if one does not exist?

Reply:
> What happens if no session exists? This would return a 404, which should probably be silenced.

If there is no session and 404 is being returned, another http post request will be issued, to create a new one